webapp/src/shared/context/auth-context.tsx

```diff
@@ -25,8 +25,8 @@ export const AuthContext = createContext<AuthContextType>(null!);
 export const AuthProvider = ({ children }: { children: ReactNode }) => {
   const [authUser, setAuthUser] = useState<User | null>(null);
   const [isLoading, setIsLoading] = useState(true);
-  const getCurrentUser = useCallback(() => {
-    return usersApi.userControllerGetCurrentUser();
+  const getCurrentUser = useCallback(async () => {
+    return await usersApi.userControllerGetCurrentUser();
```

Comment:
Why?

Reply:
More leftovers of some test I did when something wasn't working as expected